pittapi/library.py

```diff
@@ -127,3 +130,38 @@ def _extract_facets(facet_fields: List[Dict[str, Any]]) -> Dict[str, List[Dict[s
             facets[facet["display_name"]].append({"value": count["value"], "count": count["count"]})
 
     return facets
+
+
+def hillman_total_reserved():
+    """Returns a simple count dictionary of the total amount of reserved rooms appointments"""
+    count = {}
+    resp = requests.get(STUDY_ROOMS_URL)
+    resp = resp.json()
+    # Total records is kept track of by default in the JSON
+    total_records = resp["recordsTotal"]
+
+    # Note: this must align with the amount of entries in reserved times function; renamed for further clarification
+    count["Total Hillman Reservations"] = total_records
+    return count
+
+
+def reserved_hillman_times():
+    """Returns a list of dictionaries of reserved rooms of the Hillman with their respective times"""
+    bookings = []
+
+    resp = requests.get(STUDY_ROOMS_URL)
+    resp = resp.json()
+    data = resp["data"]
+
+    if data is None:
+        return bookings
+
+    # Note: there can be multiple reservations in the same room, hence why we must use a list of maps, and cannot just use a singular map
+    for reservation in data:
+        bookings.append(
+            {
+                "Room": reservation["itemName"],
+                "Reserved": [reservation["from"], reservation["to"]],
+            }
+        )
+    return bookings
```

Comment:
```suggestion
    resp = requests.get(STUDY_ROOMS_URL)
    resp = resp.json()
    data = resp["data"]

    if data is None:
        return []

    # Note: there can be multiple reservations in the same room, hence why we must use a list of maps, and cannot just use a singular map
    bookings = [
        {
            "Room": reservation["itemName"],
            "Reserved": [reservation["from"], reservation["to"]],
        }
        for reservation in data
    ]
    return bookings
```
The for-loop at the end here can be written as a list comprehension. Most of the time, it's more readable and more efficient in Python to use a comprehension rather than manually iterating over a collection and appending things.